packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -63,9 +63,18 @@ const LocationFinder: React.FC<LocationFinderProps & MatomoTrackEventProps> = ({
   const [autoSuggestValue, setAutoSuggestValue] = useState<string>("");
   const [touched, setTouched] = useState<{ [key: string]: boolean }>({});
 
+  const graphQlHouseNumber = houseNumberFull
```

Comment:
Can you add a comment what you're doing here?

Reply:
I agree, not totally clear, maybe extract as a util as well. Test it too.